src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -228,12 +228,7 @@ public string NicknameFromFunctionSignatureHint(string functionSignature)
                 newName = priorNameHints[qualifiedFunction].UpgradeName;
             }
 
-            splitted = newName.Split('.');
-
-            if (splitted.Length < 2)
-                return null;
-
-            return splitted[splitted.Length - 2] + "." + splitted[splitted.Length - 1];
+            return newName;
```

Comment:
Built in nodes don't have namespace, so they don't have "." and can't be splitted.